### PR TITLE
feat(weather): rewrite to wttr.in

### PR DESCRIPTION
**Pull request by kyranet**
### TODO

- [x] Discriminate HTTP errors and send contextful messages.
- [x] Increase the contrast in Sunny and Clear cards.
- [x] Fix missing name when input is `Taumatawhakatangihangakoauauotamateaturipukakapikimaungahoronukupokaiwhenuakitanatahu`.

### Screenshots

image
image
image

### CODE REVIEW

File: src/commands/Google/weather.ts [hunk 1]
```diff
@@ -40,218 +30,148 @@ const flags = ['fahrenheit', 'f', 'imperial', 'i'];
...
+		const halfIconSize = iconSize / 2;
+		const iconMargin = iconSize + 10;
+
+		return (
+			new Canvas(coordinates.width, coordinates.height)
+				.save()
+				.setShadowColor('rgba(0,0,0,.7)')
+				.setShadowBlur(coordinates.margin / 2)
+				.setColor(background)
+				.createRoundedPath(coordinates.margin, coordinates.margin, coordinates.cardWidth, coordinates.cardHeight, coordinates.margin / 2)
```
Comment (vladfrangu) [inline src/commands/Google/weather.ts]: any reason you don't deconstruct these all

File: src/languages/en-US/commands/google.json [hunk 2]
```diff
@@ -80,15 +80,15 @@
 			"Place",
 			"Place --imperial/--fahrenheit"
 		],
-		"extendedHelp": "This command uses Google Maps to get the coordinates of the place.\nOnce this command got the coordinates, it queries DarkSky to retrieve information about the weather.",
+		"extendedHelp": "This command uses [wttr.in](https://wttr.in) to get weather data from a location and displays it with a pretty card.",
 		"explainedUsage": [
 			[
 				"Place",
-				"The locality, governing, country, or continent to check the weather from."
+				"The place, city, or country to check the weather from."
```
Comment (vladfrangu) [inline src/languages/en-US/commands/google.json]:
```suggestion
				"The coordinates, place, city, or country to check the weather from."
```

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/languages/en-US/commands/google.json [hunk 3]
```diff
@@ -88,10 +89,10 @@
 			]
 		],
 		"examples": [
-			"Madrid",
-			"Arizona"
+			"New York",
+			"San Francisco"
 		],
-		"reminder": "Temperature is in **Celsius** by default. Use the --imperial or --fahrenheit flag to view it in **Fahrenheit**."
+		"reminder": "The units are in the Imperial System by default. Use the --celsius or --metric flag to view it in the **International System**."
```
Comment (vladfrangu) [inline src/languages/en-US/commands/google.json:95]: No? The units are in whatever system the place uses

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/languages/en-US/commands/google.json (hunk 3 above)
Comment (kyranet) [reply src/languages/en-US/commands/google.json:95]: Yes, and it's default for the US.

**Pull request merged**